src/views/Members.vue

```diff
@@ -0,0 +1,193 @@
+<template>
+  <div>
+    <section>
+      <div class="parallax" id="plax_9">
+        <h1>The Contributing Members</h1>
+        <nav>
+          <button class="nav-item cont" @click="changeView('1')">Eboard</button>
+          <button class="nav-item cont" @click="changeView('2')">
+            Development
+          </button>
+          <button class="nav-item cont" @click="changeView('3')">
+            Testing & Security
+          </button>
+          <button class="nav-item cont" @click="changeView('4')">Design</button>
+        </nav>
+      </div>
+    </section>
+
+    <section v-show="isEboard">
+      <div class="content">
+        <div class="grid-layout">
+          <Person v-for="person in people" :key="person.id" :person="person" />
+        </div>
+      </div>
+    </section>
+
+    <section v-show="isDev">
+      <div class="content">
+        <div class="grid-layout-4">
+          <Person v-for="person in people" :key="person.id" :person="person" />
+        </div>
+      </div>
+    </section>
+
+    <section v-show="isTS">
+      <div class="content">
+        <div class="grid-layout-4">
+          <Person v-for="person in people" :key="person.id" :person="person" />
+        </div>
+      </div>
+    </section>
+
+    <section v-show="isDesign">
+      <div class="content">
+        <div class="grid-layout-4">
+          <Person v-for="person in people" :key="person.id" :person="person" />
+        </div>
+      </div>
+    </section>
+  </div>
+</template>
+
+<script>
+import Person from "../components/Person";
+import { mapState, mapActions } from "vuex";
+
+export default {
+  name: "eboard",
+
+  components: {
+    Person
+  },
+  data() {
+    return {
+      isEboard: true,
+      isDev: false,
+      isTS: false,
+      isDesign: false
+    };
+  },
+
+  mounted() {
+    this.FETCH_BOARD();
+    //makes it so that when taken to page from router, it goes to the top
+    window.scrollTo(0, 0);
+  },
+
+  computed: {
+    ...mapState(["posts"]),
+    people() {
+      return this.$store.getters.GET_BOARD;
+    }
+  },
+
+  methods: {
+    ...mapActions(["FETCH_BOARD"]),
+
+    changeView: function(num) {
```

Comment:
I feel like there's a better way to do this. 

How about we have a "currentPage" field that uses strings in ["eboard", "dev", "ts", "design"] so we don't have to have all of this logic?